### DISCUSSION TITLE
Uninstall issues

### DISCUSSION THREAD

**Discussion by Taking23** [valinet/ExplorerPatcher#2069] [General]
So i found an error tired to uninstall it but it WONT REMOVE, can someone please find a solution?

Update: New update fixes the issue of what i was experiencing before, im closing this discussion

**Comment by Poopooracoocoo** [comment]
https://github.com/valinet/ExplorerPatcher/wiki#uninstalling

Others have said that deleting EP's registry keys have helped to fully uninstall EP

**Reply by Taking23** [reply]
do you know where it is, like the name of the folder?

**Reply by Poopooracoocoo** [reply]
https://github.com/valinet/ExplorerPatcher/issues/2051#issuecomment-1718734451

**Comment by DavidFeng-8844** [comment]
I found that after Installing the ExplorerPatcher,  a series of problem occurs like the Taskbar show the start icon but it can't be clicked and I even can't access to the search mode even typing the windows button, So I decided to delete it. YOU can just type WIN + R and serach fot control panel and  search for programme and Features. Right there you can find the uninstalling button

**Reply by pyrates999** [reply]
Symbols weren't downloaded.  See here for more information on that: https://github.com/valinet/ExplorerPatcher/wiki/Symbols
